pegasus/forms/hoc_signup_2014.rb

```diff
@@ -26,6 +26,28 @@ def self.normalize(data)
     result[:school_city_s] = stripped data[:school_city_s]
     result[:school_state_s] = stripped data[:school_state_s]
     result[:school_zip_s] = stripped data[:school_zip_s]
+
+    #If the user has an in-school US event, they will fill out 2017 census questions.
+    result[:role_s] = data[:role_s]
```

Comment:
Seems like we could define an array of values and put this in a loop?  (Varying for `stripped` or not, of course)